README.rst

```diff
@@ -131,10 +131,10 @@ Integration
 -----------
 
 PyPIM can be integrated in PyAnsys libraries to transparently switch to a remote
-instance in a suitable environment. This process is described in the integration
-guide.
+instance in a suitable environment. This process is described in the
+:ref:`integration` topic.
```

Comment:
```suggestion
integration topic.
```

Unfortunately this syntax is not valid for pypi, only for sphinx, and this file is used for both. Removing the link